README.md

```diff
@@ -158,23 +160,131 @@ is different. See each providers `help` command for the available actions.
 images without making them public).
 
-## plume release
+### plume release
 Publish a new Container Linux release. This makes the images uploaded by pre-release public and uploads
 images that pre-release could not. It copies the release artifacts to public storage buckets and updates
 the directory index.
 
-### plume index
+#### plume index
 Generate and upload index.html objects to turn a Google Cloud Storage
 bucket into a publicly browsable file tree. Useful if you want something
 like Apache's directory index for your software download repository.
 Plume release handles this as well, so it does not need to be run as part of
 the release process.
+
+## Platform Credentials
+Each platform reads the credentials it uses from different files. The `aws`, `esx`, `oci` and `packet`
+platforms support selecting from multiple configured credentials, call "profiles". The examples below
+are for the "default" profile, but other profiles can be specified in the credentials files and selected
+via the `--<platform-name>-profile` flag:
+```
+kola spawn -p aws --aws-profile other_profile
+```
+
+### aws
+`aws` reads the `~/.aws/credentials` file used by Amazon's aws command-line tool.
+It can be created using the `aws` command:
+```
+$ aws configure
+```
+To configure a different profile, use the `--profile` flag
+```
+$ aws configure --profile other_profile
+```
+
+or be populated manually:
+```
+[default]
+aws_access_key_id = ACCESS_KEY_ID_HERE
+aws_secret_access_key = SECRET_ACCESS_KEY_HERE
+```
+
+To install the `aws` command in the SDK, run:
+```
+sudo emerge --ask awscli
+```
+
+### azure
+TBD (FIXME)
+
+### do
+`do` uses `~/.config/digitalocean.json`. This can be configured manually:
+```
+{
+    "default": {
+        "token": "token goes here"
+    }
+}
+```
+
+### esx
+`esx` uses `~/.config/esx.json`. This can be configured manually:
+```
+{
+    "default": {
+        "server": "server.address.goes.here",
+        "user": "user.goes.here",
+        "password": "password.goes.here"
+    }
+}
+```
+
+### gce
+`gce` uses the `~/.boto` file. When the `gce` platform is first used, it will print
+a link that can be used to log into your account with gce and get a verification code
+you can paste in. This will populate the `.boto` file.
+
+See [Google Cloud Platform's Documentation](https://cloud.google.com/storage/docs/boto-gsutil)
+for more information about the `.boto` file.
+
+### oci
+`oci` reads the `~/.oci/config` file used by Oracle's oci command-line tool in addition to a
+`~/.oci/config.mantle` file. The `~/.oci/config` file can be created using the `oci` command:
+```
+oci setup config
+```
+
+or populated manually:
+```
+[DEFAULT]
+user=user.ocid.goes.here
+fingerprint=api.key.fingerprint.goes.here
+key_file=/path/to/oci_api_key.pem
+tenancy=tenancy.ocid.goes.here
+region=region-goes-here
+```
+
+The `oci setup config` command will generate the key file automatically, but it can also be
+generated via the `oci` command:
+```
+oci setup keys
```

Comment:
This seems confusing and unnecessary; maybe we should drop it.

Reply:
maybe just "The `oci setup config` command will generate the key file automatically." and drop the rest? Otherwise its unclear where the key file comes from.

Reply:
👍